### ISSUE TITLE
CSFF中xy和论文图4不符

### ISSUE THREAD

**Issue by yeqian1961** [yuliu316316/MSCAF-COD#1]
作者您好，我在拜读您的论文时发现以下疑问：
![image](https://user-images.githubusercontent.com/58923780/228158366-a6dc796e-eb66-470c-9897-454be18ff95c.png)
这里xy = self.cat2(torch.cat((x, y), dim=1)) + y + x是不是和图里不太对应
    def forward(self, x, y, edge):
        xy = self.cat2(torch.cat((x, y), dim=1)) + y + x
请指教，不甚感激
